ui/admin/app/components/form/onboarding/index.hbs

```diff
@@ -0,0 +1,63 @@
+<Rose::Form class='full-width' @cancel={{@cancel}} @onSubmit={{this.atSubmit}}>
+  <div class='create-resources-form'>
+    <Hds::Form::TextInput::Field
+      name='targetAddress'
+      @type='text'
+      @value={{this.targetAddress}}
+      @isRequired={{true}}
+      @isInvalid={{this.targetAddressInValid}}
+      {{on 'input' (set-from-event this 'targetAddress')}}
+      as |F|
+    >
+      <F.Label>{{t 'onboarding.add-target.titles.target-address'}}</F.Label>
+      <F.HelperText>
+        {{t 'onboarding.add-target.descriptions.target-address'}}
+      </F.HelperText>
+
+      {{#if this.targetAddressInValid}}
+        <F.Error as |E|>
+          <E.Message>{{t 'errors.required-field'}}</E.Message>
+        </F.Error>
+      {{/if}}
+    </Hds::Form::TextInput::Field>
+
+    <Hds::Form::TextInput::Field
+      name='targetPort'
+      @type='text'
+      @value={{this.targetPort}}
+      @isRequired={{true}}
+      @isInvalid={{this.targetPortInValid}}
+      {{on 'input' (set-from-event this 'targetPort')}}
+      as |F|
+    >
+      <F.Label>{{t 'onboarding.add-target.titles.target-port'}}</F.Label>
+      <F.HelperText>
+        {{t 'onboarding.add-target.descriptions.target-port'}}
+      </F.HelperText>
+      {{#if this.targetPortInValid}}
+        <F.Error as |E|>
+          <E.Message>{{t 'errors.required-field'}}</E.Message>
+        </F.Error>
+      {{/if}}
+    </Hds::Form::TextInput::Field>
+  </div>
+
+  <Hds::Form::Fieldset as |F|>
+    <F.Legend>{{t 'onboarding.add-target.titles.target-setup-info'}}</F.Legend>
+    <F.Control>{{t
+        'onboarding.add-target.descriptions.target-setup-info'
+      }}</F.Control>
+    <F.Control><Image @name='ssh-target-creation' /></F.Control>
+  </Hds::Form::Fieldset>
+
+  <Hds::ButtonSet>
+    <Hds::Button @text='{{t "actions.save"}}' type='submit' />
+    <Hds::Button
```

Comment:
@ZedLi Since we refactor the form validation to happen at submit time rather than on input events, I delete the submit button disable functionality. Now submit button is always ON.